app/services/campaign_page.rb

```diff
@@ -0,0 +1,24 @@
+class CampaignPage
+  attr_reader :utm_content_code, :criteria, :banner_image
+
+  def self.exists?(utm_content)
+    Rails.application.config.campaign_pages.key?(utm_content.to_sym)
+  end
+
+  def self.[](utm_content_code)
+    raise "No such campaign page: '#{utm_content_code}'" unless exists?(utm_content_code)
```

Comment:
Shouldn't we gracefully treat this so users don't get a service error when the param value is wrong?
I would expect a user to get a `404` page or get redirected to the generic vacancies page.